rotkehlchen/chain/ethereum/names.py

```diff
@@ -148,6 +149,10 @@ def _blockchain_address_to_name(
     """Returns the label of an ethereum blockchain account with the given address or
     None if there is no such account or the account has no label set.
     """
+    if chain_address.blockchain is None:
```

Comment:
Update the docstring of the function. It should evm account and you should also explain the behaviour when not specifying chain.